frontend/src/components/controls/RisDateInput.spec.ts

```diff
@@ -49,8 +69,19 @@ describe("DateInput", () => {
   })
 
   test("emits model update event when input completed and valid", async () => {
+    const CustomInputMaskStub = {
+      props: ["modelValue"],
+      template: `<input
+    :value="modelValue"
```

Comment:
Suggestion: use `<input v-model="modelValue" />` instead. This gives you autocomplete and will detect issues when we upgrade Vue. Same for other stub components in this file.